### ISSUE TITLE
brew upgrade emerald-cli: Error: No such file or directory - emerald

### ISSUE THREAD

**Issue by whilei** [ethereumproject/homebrew-classic#1]
```
~  ⟠ brew upgrade ethereumproject/classic/emerald-cli

[...]

==> Upgrading 1 outdated package, with result:
ethereumproject/classic/emerald-cli 0.19.0 -> 0.21.0
==> Upgrading ethereumproject/classic/emerald-cli
==> Downloading https://github.com/ETCDEVTeam/emerald-cli/releases/download/v0.21.0/emerald-cli-v0.21.0-OSX-x86_64.tar.gz
Already downloaded: /Users/ia/Library/Caches/Homebrew/emerald-cli-0.21.0.tar.gz
Error: No such file or directory - emerald
```

**Comment by whilei** [comment]
I think because binary is nested in folder `target/release/`

```
-rw-r--r--  1 ia staff  68K Apr  4 13:32 emerald-cli-v0.21.0-OSX-x86_64.tar.gz

[expand .tar.gz...]

drwxr-xr-x  3 ia staff  102 Apr  6 09:56 target/
~/sandbox  ⟠ cd target/
~/sandbox/target  ⟠ l
total 0
drwxr-xr-x 3 ia staff 102 Apr  6 09:56 release/
~/sandbox/target  ⟠ cd release/
~/sandbox/target/release  ⟠ l
total 13M
-rwxr-xr-x 1 ia staff 13M Apr  3 05:46 emerald
```

**Comment by whilei** [comment]
resolved as of 6c78c8a
